modules/ugdk-core/include/ugdk/structure/vertex.h

```diff
@@ -1,21 +1,36 @@
+
 #ifndef UGDK_STRUCTURE_VERTEX_H_
 #define UGDK_STRUCTURE_VERTEX_H_
 
+#include <ugdk/structure/types.h>
+
+#include <glm/vec2.hpp>
+
 namespace ugdk {
 namespace structure {
 
 struct VertexXYUV {
-    float x;
-    float y;
-    float u;
-    float v;
+
+    F32 x;
```

Comment:
What's the advantage of using this type alias?

Reply:
Knowing the type size. And it's shorter than `float32_t`. The exact wording was based on the "game engine architecture" book.